schemas/fb-linux.yml

```diff
@@ -44,13 +47,24 @@
       os_version:
         - 2
 
+- src: "{app_name}-{version}-1.amazonlinux-{os_version}.{arch}.rpm"
```

Comment:
what is the difference between `fb-linux.yml` and `fb-amazonlinux.yml`? Why have both?

Reply:
fb-linux is an all-in-one schema, so it contains all the linux distributions we want to release packages for.
fb-amazonlinux is only for amazonlinux, we can use those from the "staged" pre-release an release github actions if we need to release the packages for only one distribution.